packages/web-components/src/components/cta-block/cta-block.ts

```diff
@@ -98,7 +98,10 @@ class C4DCTABlock extends StableSelectorMixin(C4DContentBlock) {
   protected _renderActions(): TemplateResult | string | void {
     const { _hasAction: hasAction, _handleSlotChange: handleSlotChange } = this;
     return html`
-      <div ?hidden="${!hasAction}" class="${prefix}--content-layout__cta">
+      <div
+        ?hidden="${!hasAction}"
+        class="${prefix}--content-layout__cta"
+        part="action">
```

Comment:
Suggesting that we stick with the element name from the class eg. `cta`.
```suggestion
        part="cta">
```